Projects/Governance/Schemas/Docs-Topics/G/Gov/Gov-Commands-Messages/gov-commands-messages-006-gov-message-automated-user-profile-contribute-done.json

```diff
@@ -0,0 +1,70 @@
+{
+    "topic": "Gov Commands Messages",
+    "pageNumber": "6",
+    "type": "Gov Message - Automated User Profile Contribute Done",
+    "definition": {
+        "text": "Your user profile was uploaded to your Superalgos Fork and a Pull Request was submitted to the main Superalgos repository.",
+        "updated": 1611002547907,
+        "translations": [
+            {
+                "language": "ID",
+                "text": "Kontribusi anda telah terupload ke Superalgos Fork dan Pull Request telah dikirim ke penyimpanan utama Superalgos.",
+                "updated": 1613861035790,
+                "style": "Definition"
+            },
+            {
+                "language": "ES",
+                "text": "Tus contribuciones se cargaron en tu Superalgos Fork y tu Pull Request fue enviado al repositorio principal de Superalgos.",
+                "updated": 1628207320785,
```

Comment:
I guess you did make this file and add all the different translations?
I have seen this phenomenon a few times with different pull requests. It somehow seems to have its origin in Superalgos.